dashboard/test/models/course_offering_test.rb

```diff
@@ -107,7 +107,7 @@ class CourseOfferingTest < ActiveSupport::TestCase
     offering = CourseOffering.add_course_offering(content_root)
 
     assert_nil offering
-    assert_nil content_root.course_version
+    assert_nil content_root.get_course_version
```

Comment:
it looks like we can delete this test case, because we should never be adding a course offering to a unit (see other comment about enforcing this via `raise`)